src/modules/account/components/AccountSelect/AccountSelect.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import Box from 'src/theme/box';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { OutlineButton } from 'src/theme/buttons';
+import Icon from 'src/theme/Icon';
+import WalletVisual from '@wallet/components/walletVisual';
+import { useAccounts } from '../../hooks/useAccounts';
+import styles from './AccountSelect.css';
+
+function AccountRow({
+  account,
+  onSelect,
+  onRemove,
+  showRemove,
+}) {
+  const { uuid, metadata: { name, address } } = account;
+
+  return (
+    <div
+      key={uuid}
+      data-testid={uuid}
+      className={styles.accountWraper}
+      onClick={() => onSelect(account)}
+    >
+      <WalletVisual address={address} size={40} />
```

Comment:
This should be more like AccountIcon? I know in the past we might have used it differently but from AccountIcon makes more relatable here?